roommate-app-testing.UnitTests/UnitTest1.cs

```diff
@@ -112,17 +119,19 @@ public void E_Login_EmailNotCorrectAndPasswordCorrect_ReturnsFalse()
         }
 
         [Fact]
-        public void F_Login_EmailAndPasswordNotCorrect_ReturnsFalse()
+        public void Login_EmailAndPasswordNotCorrect_ReturnsFalse()
         {
             // Arrange
-            var options = new DbContextOptionsBuilder<ApplicationDbContext>().UseInMemoryDatabase("LocalDatabase").Options;
+            var options = new DbContextOptionsBuilder<ApplicationDbContext>().UseInMemoryDatabase(Guid.NewGuid().ToString()).Options;
             var _dbContext = new ApplicationDbContext(options);
             var genericService = new GenericService(_dbContext);
             var file = new FileCreator();
             var errorLoging = new ErrorLogging(file);
             var loginController = new LoginController(errorLoging, genericService);
+            var registrationController = new RegistrationController(genericService);
 
             // Act
+            var submit = registrationController.Submit(new User { City = "", FirstName = "", LastName = "", Email = "abc@abc.com", Password = "password" });
```

Comment:
What's the point of this submit variable? I don't see it being used anywhere in this test.